drivers/ethernet/Kconfig

```diff
@@ -8,19 +8,13 @@
 
 menu "Ethernet Drivers"
 
-config SYS_LOG_ETHERNET_LEVEL
-	int "Ethernet driver log level"
-	depends on SYS_LOG && NET_L2_ETHERNET
-	default 0
-	range 0 4
-	help
-	  Sets log level for Ethernet Device Drivers.
-	  Levels are:
-	  0 OFF, do not write
-	  1 ERROR, only write SYS_LOG_ERR
-	  2 WARNING, write SYS_LOG_WRN in addition to previous level
-	  3 INFO, write SYS_LOG_INF in addition to previous levels
-	  4 DEBUG, write SYS_LOG_DBG in addition to previous levels
+if NET_L2_ETHERNET
+module=ETHERNET
```

Comment:
I think we should set syntactic conventions on how to specify template params right away, and I'd say that without spaces around "=" these don't look pretty.

Actually, we already have ` = ` in the codebase, e.g. subsys/shell/Kconfig, so let's follow that.

cc: @ulfalizer

Reply:
Indeed I seem to have missed couple of these template changes in drivers. I will send a PR shortly for these.